CODE_OF_CONDUCT.md

```diff
@@ -1,6 +1,36 @@
+### Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+  - [Positive participation and communication](#positive-participation-and-communication)
+  - [Prohibited Behavior](#prohibited-behavior)
+    - [Discrimination](#discrimination)
+    - [Harassment and privacy violation](#harassment-and-privacy-violation)
+    - [Harmful and/or adult content](#harmful-andor-adult-content)
+  - [Where does this CoC apply and who does this CoC apply to?](#where-does-this-coc-apply-and-who-does-this-coc-apply-to)
+  - [What if someone violates this Code of Conduct while outside of Florence instances, or in a space or medium to which the Code does not apply?](#what-if-someone-violates-this-code-of-conduct-while-outside-of-florence-instances-or-in-a-space-or-medium-to-which-the-code-does-not-apply)
+  - [When Something Happens](#when-something-happens)
+    - [Example](#example)
+  - [How to make a Code Of Conduct Report](#how-to-make-a-code-of-conduct-report)
+    - [Information to include in your report](#information-to-include-in-your-report)
+  - [Code of Conduct Reporting Guide](#code-of-conduct-reporting-guide)
+  - [What happens after you file a report?](#what-happens-after-you-file-a-report)
+    - [Example](#example-1)
+  - [What if your report concerns a possible violation by a moderator?](#what-if-your-report-concerns-a-possible-violation-by-a-moderator)
+  - [How CoC Reports are handled by moderators](#how-coc-reports-are-handled-by-moderators)
+  - [Accountability for board members and moderators](#accountability-for-board-members-and-moderators)
+  - [Definition of Terms](#definition-of-terms)
+  - [Reference Codes of Conduct](#reference-codes-of-conduct)
+  
 # Code of Conduct
 
-## Positive participation and communication 
+We strive to reduce barriers to access of our organisation, our software and our community, as well as to dismantle any ableist assumptions and practices in our process. We recognize that there is no one solution for universal accessibility, and strive to provide alternatives and choices whenever possible.
```

Comment:
If you are using Americanisms it's important to stick to them. "Organisation" should be "Organization"